mcp/README.md

```diff
@@ -0,0 +1,55 @@
+# Model Context Protocol (MCP) Servers
+
+This directory contains information about Model Context Protocol (MCP) servers that extend AI capabilities by providing tools and resources for interacting with external systems and APIs.
+
+## Available MCP Servers
+
+| Repository | Description | Features/Tools | Installation | Integration |
+|------------|-------------|----------------|-------------|-------------|
+| [mongodb-developer/mcp-mongodb-atlas](https://github.com/mongodb-developer/mcp-mongodb-atlas) | An MCP server for managing MongoDB Atlas projects. Provides tools for creating and managing MongoDB Atlas clusters, users, and network access through the MCP interface. | • `create_atlas_cluster` - Create a new MongoDB Atlas cluster<br>• `setup_atlas_network_access` - Configure network access<br>• `create_atlas_user` - Create a new database user<br>• `get_atlas_connection_strings` - Retrieve connection strings<br>• `list_atlas_projects` - List all Atlas projects<br>• `list_atlas_clusters` - List all clusters in a project | `npm install mcp-mongodb-atlas` | **VSCode (Cline)**:<br>Add to `cline_mcp_settings.json`<br><br>**Cursor**:<br>Add to MCP settings or `~/.cursor/mcp.json`<br><br>**Claude Desktop**:<br>Add to `claude_desktop_config.json` |
+
+## Usage Examples
+
+### MongoDB Atlas MCP Server
+
+#### Command Line Usage
+```bash
+# Using environment variables
+export ATLAS_PUBLIC_KEY="your-public-key"
```

Comment:
Can you add some instructions on how to get these?

Reply:
Done